src/Documentation/Advanced-Concepts/Heterogeneous-Silos.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Heterogeneous silos
+---
+
+# Heterogeneous silos
+On a given cluster, silos can support a different set of grain types:
```

Comment:
Is Heterogeneous.png used? Could we also checkin the src file for it (assuming it was generated from some other file)?